lib/Conversion/TorchToLinalg/Uncategorized.cpp

```diff
@@ -528,10 +534,29 @@ static Value createLinalgPayloadCalculationForElementwiseOp(
     return convertScalarToDtype(b, loc, div, outTy, std::nullopt, outTTy);
   }
   if (auto relu = dyn_cast<AtenReluOp>(op)) {
-    if (!relu.getType()
-             .cast<ValueTensorType>()
-             .getDtype()
-             .isa<mlir::FloatType>()) {
+    Value zeroPoint;
+    getZeroPoint(relu.getSelf(), zeroPoint);
+    Value arg = payloadArgs[0];
+    auto reluTorchType = cast<ValueTensorType>(relu.getType());
+    if (zeroPoint) {
```

Comment:
This is better if it is

```
if (!zeroPoint) {
      relu.emitError("unimplemented: non-integer quantized Relu.");
      return nullptr;
}
```

Also move up to where you get the zero point value